packages/neuron-ui/src/components/WalletWizard/index.tsx

```diff
@@ -193,8 +196,9 @@ const Mnemonic = ({ state = initState, rootPath = '/wizard/', dispatch }: Wizard
     [MnemonicAction.Verify]: 'wizard.replenish-your-seed',
     [MnemonicAction.Import]: 'wizard.input-your-seed',
   }[type]
-  const { inputsWords, onChangeInput, setInputsWords } = useInputWords()
   const [searchParams] = useSearchParams()
+  const isHardware = searchParams.get('isHardware') === 'true'
```

Comment:
it would be clearer here to use `wordCount` as the `props` because  the 24-word-mnemonic feature is exclusive for hardware wallets

Reply:
I guess `isHardware` is better, then we know the `hardware` wallet should import 24 words.